modules/nw-configuring-sriov-rdma-cni.adoc

```diff
@@ -0,0 +1,177 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-rdma-cni.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-sriov-configuring-sriov-rdma-cni_{context}"]
+= Configuring SR-IOV RDMA CNI
+
+You can configure an RDMA CNI on SR-IOV.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. Create an `SriovNetworkPoolConfig` CR and save it as `sriov-nw-pool.yaml`, as in the following example:
++
+.Example `SriovNetworkPoolConfig` CR
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkPoolConfig
+metadata:
+  name: worker
+  namespace: openshift-sriov-network-operator
+spec:
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/workercnf: ""
+  rdmaMode: exclusive <1>
+----
+<1> Set RDMA network namespace mode to `exclusive`.
+
+. Create the `SriovNetworkPoolConfig` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-nw-pool.yaml
+----
+
+. Create an `SriovNetworkNodePolicy` CR and save it as `sriov-node-policy.yaml`, as in the following example: 
++
+.Example `SriovNetworkNodePolicy` CR
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-nic-pf1
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  isRdma: true <1>
+  nicSelector:
+    pfNames: ["ens3f0np0"]
+  nodeSelector:
+    node-role.kubernetes.io/workercnf: ""
+  numVfs: 4
+  priority: 99
+  resourceName: sriov_nic_pf1
+----
+<1> Activate RDMA mode.
+
+. Create the `SriovNetworkNodePolicy` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-node-policy.yaml
+----
+
+. Create an `SriovNetwork` CR and save it as `sriov-network.yaml`, as in the following example: 
++
+.Example `SriovNetwork` CR
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-nic-pf1
+  namespace: openshift-sriov-network-operator
+spec:
+  networkNamespace: sriov-tests
+  resourceName: sriov_nic_pf1
+  capabilities: '{ "mac": true, "ips": true }'
+  ipam: '{ "type": "static" }'
```

Comment:
let have it empty link https://docs.openshift.com/container-platform/4.17/networking/hardware_networks/configuring-sriov-net-attach.html#nw-sriov-network-object_configuring-sriov-net-attach